src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/AprilTagCamera.java

```diff
@@ -0,0 +1,186 @@
+package frc.trigon.robot.poseestimation.apriltagcamera;
+
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.*;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+import frc.trigon.robot.Robot;
+import frc.trigon.robot.constants.FieldConstants;
+import frc.trigon.robot.poseestimation.poseestimator.PoseEstimator6328;
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonUtils;
+
+/**
+ * An april tag camera is a class that provides the robot's pose, from a camera.
+ */
+public class AprilTagCamera {
+    protected final String name;
+    private final RobotPoseSourceInputsAutoLogged inputs = new RobotPoseSourceInputsAutoLogged();
+    private final Transform3d robotCenterToCamera;
+    private final AprilTagCameraIO aprilTagCameraIO;
+    private double lastUpdatedTimestamp;
+    private final double solvePNPTranslationsStdExponent, assumedRobotPoseTranslationsStdExponent, solvePNPThetaStdExponent;
+    private Pose2d robotPose = null;
+
+    /**
+     * @param robotPoseSourceType                        the type of camera
+     * @param name                                       the camera's name
+     * @param robotCenterToCamera                        the transform of the robot's origin point to the camera
+     * @param solvePNPTranslationsStdExponent            the translation exponent for solve PNP
+     * @param solvePNPThetaStdExponent                   the theta exponent for solve PNP
+     * @param assumedRobotHeadingTranslationsStdExponent the translation exponent for the assumed robot heading pose calculation
+     */
```

Comment:
Your javadoc doesn't really add anything. Explain more and better.

Reply:
How?

Reply:
Wdym?

Reply:
How can I add more? I don't know what else to write.

Reply:
For starters, what's std devs, and more
Whatever you think you wouldn't. understand if this was the first time you were doing vision. 
I don't want to see stddevs anywhere in docs btw, always write Standard Deviations (and maybe add brackets and write `Standard Deviations (StdDevs)`